deployer/steps.go

```diff
@@ -126,16 +126,8 @@ func (d *Deployer) StepGenRawDisk() error {
 func (d *Deployer) StepGenMBRRawDisk() error {
 	return d.Add(opGenMBRRawDisk,
 		herd.EnableIf(func() bool { return d.Config.Disk.ARM == nil && d.Config.Disk.MBR }),
-		herd.IfElse(d.isoOption(),
-			herd.WithDeps(opDownloadISO), herd.WithDeps(opGenISO),
-		),
-		herd.IfElse(d.isoOption(),
-			herd.WithCallback(
-				ops.GenBIOSRawDisk(d.Config, d.isoFile(), d.rawDiskPath())),
-			herd.WithCallback(
-				ops.GenBIOSRawDisk(d.Config, d.isoFile(), d.rawDiskPath())),
-		),
-	)
+		d.imageOrSquashFS(),
```

Comment:
Im not sure if this si correct. I.E. are we trying to create a BIOS image vs an EFI image or a MBR raw image VS a GPT raw image?

Notice that an MBR image can only support BIOS but an GPT image can support both BIOS and EFI. 

I think this was meant as BIOS image so the notation may be wrong or my approach is wrong :D